src/ansys/dpf/post/result_data.py

```diff
@@ -322,7 +323,12 @@ def _sort_fields_container_with_labels(self, option_id, display_option):
             new_fields_container.add_field(label, field)
         return new_fields_container
 
-    def plot_contour(self, display_option: str = "time", option_id=1, **kwargs):
+    def plot_contour(
+        self,
+        display_option: Optional[str] = "time",
```

Comment:
@GuillemBarroso quick question: is `Optional` mandatory even when it obviously is due to the default value?

Reply:
It's not mandatory! I was just trying to follow the same criteria. You can add Optional whenever you want. For instance, this source code:

![image](https://user-images.githubusercontent.com/53979143/193299492-2f04f1c5-6cc6-450c-918e-4cad5070b227.png)


Renders as:

![image](https://user-images.githubusercontent.com/53979143/193299399-32e35c8d-af2f-454e-a5f5-c377bc68886e.png)

Is that what you meant?

Reply:
See also: https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
True! That's what we discussed during last meeting. I have amended the method's description without ``typing.Optional`` so it know renders as:

![image](https://user-images.githubusercontent.com/53979143/193307956-2b29e793-bd4f-4fcc-a880-97df4143cf30.png)

I will change the optional parameters depending on the poll's result.

Reply:
> I will change the optional parameters depending on the poll's result.

If `braces-after` ends up winning, we should first figure out if we can get around that bug.